brownie-config.yaml

```diff
@@ -10,3 +10,17 @@ compiler:
     remappings:
       - "@openzeppelin=OpenZeppelin/openzeppelin-contracts@4.5.0"
       - "@openzeppelin-upgradeable=OpenZeppelin/openzeppelin-contracts-upgradeable@4.5.0"
+networks:
+  default: development
+  rinkeby:
+    nu_token: '0x78D591D90a4a768B9D2790deA465D472b6Fe0f18'
+    t_staking: '0x18eFb520dA5D387982C860a64855C14C0AcADF3F'
+    work_lock: '0x0000000000000000000000000000000000000000'
+    verify: True
+  mainnet:
+    nu_token: '0x4fE83213D56308330EC302a8BD641f1d0113A4Cc'
+    t_staking: '0x01B67b1194C75264d06F808A921228a95C765dd7'
+    work_lock: '0xe9778E69a961e64d3cdBB34CF6778281d34667c2'
+    verify: True
+  development:
+    verify: False
```

Comment:
```suggestion
    verify: False

```

Reply:
Yes, you can define your own networks. So we could easily have `rinkeby (ibex)` and `rinkeby (lynx?)`.

These would then have their own configs defined pointing at different instances of the contracts.